src-script/pack-apack-win-linux.js

```diff
@@ -6,12 +6,14 @@ exports.default = async function (context) {
     context.electronPlatformName === 'win32' ||
     context.electronPlatformName === 'linux'
   ) {
-    return scriptUtil.executeCmd({}, 'npx', [
+    await scriptUtil.executeCmd({}, 'npx', [
```

Comment:
Why did we need to await here?

Reply:
to wait for it to resolve for console.log